ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/dot1x.schema.yml

```diff
@@ -147,3 +147,21 @@ keys:
             min: 5
             max: 65535
             description: Interval period in seconds.
+      captive_portal:
+        type: dict
+        description: Web authentication feature that authenticates a supplicant through a web page, referred to as a captive portal.
+        keys:
+          url:
+            type: str
+          ssl_profile:
+            type: str
+            description: SSL profile name.
+          access_list_ipv4:
+            type: str
+            description: Standard access-list name.
+      supplicant:
+        type: dict
```

Comment:
We are keeping this dict for future use I believe.

Reply:
yes